pkg/sriov/sriov.go

```diff
@@ -272,6 +272,17 @@ func getVfInfo(link netlink.Link, id int) *netlink.VfInfo {
 // ApplyVFConfig configure a VF with parameters given in NetConf
 func (s *sriovManager) ApplyVFConfig(conf *sriovtypes.NetConf) error {
 
+	// It is not possible to setup VF config if there is no PF associated with the VF, so check if there is any
+	// config and reject if the master has not been set.
+	if conf.Master == "" {
+		if conf.Vlan != 0 || conf.MAC != "" || conf.MinTxRate != nil || conf.MaxTxRate != nil || conf.SpoofChk != "" || conf.LinkState != "" {
```

Comment:
@cswindle It is not possible to configure MAC address or VLAN to VF with no access to its PF?